CHANGES.md

```diff
@@ -1,13 +1,25 @@
 ## In the next release
 
+## 0.0.3
+
+* New admin endpoint! POST to `/shutdown` to tell the process to exit.
+* Ensure that remote streams are shutdown properly.
```

Comment:
Could word this slightly differently:

* Fix connection leak when one peer disconnected

or something like that